src/platform.ts

```diff
@@ -107,52 +120,48 @@ function getRuntimeIdLinux(distributionName: string, distributionVersion: string
     switch (distributionName) {
         case 'ubuntu':
             if (distributionVersion.startsWith('14')) {
-                // This also works for Linux Mint
                 return Runtime.Ubuntu_14;
             } else if (distributionVersion.startsWith('16')) {
                 return Runtime.Ubuntu_16;
+            } else if (distributionVersion.startsWith('18')) {
+                return Runtime.Ubuntu_18;
+            } else if (distributionVersion.startsWith('20')) {
+                return Runtime.Ubuntu_20;
+            } else if (distributionVersion.startsWith('22')) {
+                return Runtime.Ubuntu_22;
+            } else {
+                return Runtime.Ubuntu;
             }
-
-            break;
         case 'elementary':
         case 'elementary OS':
             if (distributionVersion.startsWith('0.3')) {
-                // Elementary OS 0.3 Freya is binary compatible with Ubuntu 14.04
-                return Runtime.Ubuntu_14;

```

Comment:
there are some fallback logics (like this one) built into this method, I changed it so that it will return the actual runtime here and the fallback logic is handled separately.